Makefile.toml

```diff
@@ -0,0 +1,68 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+TARGET = { value = "${CARGO_MAKE_RUST_TARGET_TRIPLE}", condition = { env_not_set = [
+    "TARGET",
+] } }
+
+[tasks.format]
+install_crate = { crate_name = "rustfmt-nightly", binary = "rustfmt", test_arg = "--help" }
+command = "cargo"
+args = ["+nightly", "fmt", "--", "--check"]
+
+[tasks.clippy]
+install_crate = "cargo-clippy"
+command = "cargo"
+args = ["clippy"]
+
+[tasks.check]
+command = "cargo"
+args = ["check"]
+
+[tasks.test]
+command = "cargo"
+args = ["test"]
+
+[tasks.ziglang]
+script = '''
+#!/usr/bin/env bash
+if ! pip3 show ziglang &> /dev/null; then
+    pip3 install ziglang
+fi
+'''
+
+[tasks.target]
+command = "rustup"
+args = ["target", "add", "${TARGET}"]
+
+[tasks.deb]
+condition = { files_exist = ["${CARGO_MAKE_CRATE_TARGET_DIRECTORY}/${TARGET}/release/${CARGO_MAKE_CRATE_NAME}"] }
+install_crate = "cargo-deb"
+command = "cargo"
+args = ["deb", "--no-strip", "--no-build", "--deb-version=${CARGO_MAKE_CRATE_VERSION}", "--target=${TARGET}"]
+
+[tasks.build]
+command = "cargo"
+args = ["build"]
+
+[tasks.release-build]
+condition = { files_modified = { input = [
+    "./crates/**/*.rs",
+    "./plugins/**/*.rs",
+    "./crates/**/*/Cargo.toml",
+    "./plugins/**/*/Cargo.toml",
+    "./Cargo.toml",
+], output = [
+    "./target/**/release/*",
+] } }
+install_crate = "cargo-zigbuild"
+command = "cargo"
+args = ["zigbuild", "--target=${TARGET}", "--release"]
+
+[tasks.release-flow]
+dependencies = ["check", "format", "clippy", "target", "ziglang", "release-build", "deb"]
+
+[tasks.build-flow]
+dependencies = ["check", "format", "clippy", "build"]
```

Comment:
I'm not sure that this task is useful as we already have `build` and `test-flow`